### PR TITLE
Incorporate feedback

### LINKED ISSUE #15: Incorporate Draft Feedback
Incorporate the feedback of the draft. The following things have to be done:

- [x] Good structure, but you shouldn’t have so many subsections for “Generative Models” and “Detector Models”. Try breaking everything into paragraphs. Also, the “Generating Models” section should be “Generative Models”.
- [x] Please include details regarding InformatiCup 2024 in your intro and slides.
- [x] Please include citations and references in both your report and your slides

### PR DESCRIPTION
**Pull request by RaphaW**
Added the given feedback, apart from the paragraph part, since I have no idea what they exactly want

@1Maxnet1 @EhrenbruderRoschtbraten @LeviUniAcc 

Closes #15

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -39,10 +39,11 @@
...
 
-Therefore researchers evaluated different approaches to detect fake images and texts. Some adversarial attacks are already known to be able to reduce the performance of other computer vision tasks, e. g. visual object detection, by artificially manipulating real images. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.
+Therefore researchers evaluated different approaches to detect fake images and texts. Some adversarial attacks are already known to be able to reduce the performance of other computer vision tasks, e. g. visual object detection, by artificially manipulating real images.
 
+As this is a very topical issue the "Gesellschaft für Informatik" \cite{GI} organizes a competition called "InformatiCup2024" \cite{InformatiCup} in which the topic is to manipulate a detectable source in a way that the detector does not recognize it anymore. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]:
```suggestion
As this is a very topical issue the "Gesellschaft für Informatik" \cite{GI} organizes a competition called "InformatiCup2024" \cite{InformatiCup} in which the topic is to manipulate a detectable source in a way that the detector does not recognize it as AI-generated anymore. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.
```

**Review by EhrenbruderRoschtbraten — COMMENTED** [review]
Only one small comment, otherwise looks fine.

File: paper.tex [hunk 2]
```diff
@@ -39,10 +39,11 @@
...
 
-Therefore researchers evaluated different approaches to detect fake images and texts. Some adversarial attacks are already known to be able to reduce the performance of other computer vision tasks, e. g. visual object detection, by artificially manipulating real images. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.
+Therefore researchers evaluated different approaches to detect fake images and texts. Some adversarial attacks are already known to be able to reduce the performance of other computer vision tasks, e. g. visual object detection, by artificially manipulating real images.
```
Comment (1Maxnet1) [inline paper.tex]: To my knowledge, e.g. is written without a space
```suggestion
Therefore researchers evaluated different approaches to detect fake images and texts. Some adversarial attacks are already known to be able to reduce the performance of other computer vision tasks, e.g. visual object detection, by artificially manipulating real images.
```

File: paper.tex [hunk 3]
```diff
@@ -55,15 +56,15 @@ \section{Introduction}
...
 \section{Related work}
 
-\subsection{Generating models}
+\subsection{Generative models}
 
-\subsubsection{Image generating models}
+\subsubsection{Image generative models}
 
-\subsubsection{Text generating models}
+\subsubsection{Text generative models}
```
Comment (1Maxnet1) [inline paper.tex]: To my understanding the feedback regarding the paragraphs is meant, that we should remove those headings and rather just separate the sections by paragraphs. So if you could remove them as well, I think all feedback would be incorporated.

File: paper.tex [hunk 4]
```diff
@@ -55,15 +56,15 @@ \section{Introduction}
...
-The report is structured as follows: After this introduction chapter 2 presents related work, especially text and image generating models as well as fake image and text detectors. Chapter 3 presents the methodology of our work followed by chapter 4 presenting the results of our experiments. Afterwards the results will be discussed in chapter 5 and chapter 6 will conclude this paper.
+The report is structured as follows: After this introduction chapter 2 presents related work, especially text and image generative models as well as fake image and text detectors. Chapter 3 presents the methodology of our work followed by chapter 4 presenting the results of our experiments. Afterwards the results will be discussed in chapter 5 and chapter 6 will conclude this paper.
 
 \section{Related work}
 
-\subsection{Generating models}
+\subsection{Generative models}
 
-\subsubsection{Image generating models}
+\subsubsection{Image generative models}
```
Comment (1Maxnet1) [inline paper.tex]:
```suggestion
```

File: paper.tex (hunk 3 above)
Comment (1Maxnet1) [inline paper.tex]:
```suggestion
```

**Review by 1Maxnet1 — COMMENTED** [review]
Left my comments, looks good in general :)

File: paper.tex (hunk 2 above)
Comment (RaphaW) [reply paper.tex]: I don't think there should be one 

https://english.stackexchange.com/questions/298507/should-there-be-a-space-in-e-g-i-e-between-a-dot

But please confirm

**Comment by RaphaW** [comment]
@1Maxnet1 the citation is working ?

File: paper.tex (hunk 2 above)
Comment (EhrenbruderRoschtbraten) [reply paper.tex]: Yeah, you can do it without space

File: paper.tex [hunk 5]
```diff
@@ -39,10 +39,11 @@
 \end{abstract}
 
 \section{Introduction}
-The development of generative Machine Learning models made huge steps forward in the last years, considering the comparison with human crafted artefacts. Large Language Models (LLM) enable the generation of texts in all kinds of different flavours and styles. Image generating models like Vision Transformer (ViT) already created fake images being reported as real by media. Other news showed artists winning art competitions with generated art. This creates an interest in detecting which texts or images are AI generated and which ones are real images or human-written texts.
+The development of generative Machine Learning models made huge steps forward in the last years, considering the comparison with human crafted artefacts. Large Language Models (LLM) enable the generation of texts in all kinds of different flavours and styles. Image generative models like Vision Transformer (ViT) already created fake images being reported as real by media. Other news showed artists winning art competitions with generated art. This creates an interest in detecting which texts or images are AI generated and which ones are real images or human-written texts.
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]:
```suggestion
The development of generative Machine Learning models made huge steps forward in the last years considering the comparison with human crafted artefacts. Large Language Models (LLM) enable the generation of texts in all kinds of different flavours and styles. Image generative models like Vision Transformer (ViT) already created fake images being reported as real by media. Other news showed artists winning art competitions with generated art. This creates an interest in detecting which texts or images are AI generated and which ones are real images or human-written texts.
```

File: paper.tex [hunk 6]
```diff
@@ -39,10 +39,11 @@
...
 
-Therefore researchers evaluated different approaches to detect fake images and texts. Some adversarial attacks are already known to be able to reduce the performance of other computer vision tasks, e. g. visual object detection, by artificially manipulating real images. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.
+Therefore researchers evaluated different approaches to detect fake images and texts. Some adversarial attacks are already known to be able to reduce the performance of other computer vision tasks, e. g. visual object detection, by artificially manipulating real images.
 
+As this is a very topical issue the "Gesellschaft für Informatik" \cite{GI} organizes a competition called "InformatiCup2024" \cite{InformatiCup} in which the topic is to manipulate a detectable source in a way that the detector does not recognize it as AI-generated anymore. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]:
```suggestion
As this is a very topical issue the "Gesellschaft für Informatik" \cite{GI} organizes a competition called "InformatiCup2024" \cite{InformatiCup} where the participants try to manipulate a detectable source in a way that the detector does not recognize it as AI-generated anymore. This raises the questions, whether and how it is possible to manipulate generated artefacts (e.g. images and texts) to reduce the accuracy of detectors.
```

File: paper.tex [hunk 7]
```diff
@@ -55,28 +56,19 @@ \section{Introduction}
...
-
-\subsubsection{Text generating models}
+\subsection{Generative models}
 
 \subsection{Detector models}
 
-\subsubsection{Image detector models}
-
-
 The \textbf{AIorNot} detector by Nahrawy on Hugging Face is a pretrained swin-tiny-patch4-window7-224 fine-tuned on the AI or Not dataset. Swin-Transformer by Microsoft is a new Vision Transformer which makes use of hierarchical windows. This reduces the self-attention to not-overlapping windows of other areas of the image \cite{liu2021swin}.
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]: I think citations chould always be after the end of the sentence
```suggestion
The \textbf{AIorNot} detector by Nahrawy on Hugging Face is a pretrained swin-tiny-patch4-window7-224 fine-tuned on the AI or Not dataset. Swin-Transformer by Microsoft is a new Vision Transformer which makes use of hierarchical windows. This reduces the self-attention to not-overlapping windows of other areas of the image. \cite{liu2021swin}
```

**Review by EhrenbruderRoschtbraten — COMMENTED** [review]
Only some small changes

File: paper.tex (hunk 2 above)
Comment (1Maxnet1) [reply paper.tex]: @RaphaW Yeah that's why I proposed to remove it ...

File: paper.tex (hunk 7 above)
Comment (1Maxnet1) [reply paper.tex]: I disagree. This does depend on what you reference. If the hole statement is based on the cited source it makes sense at the end of the sentence, but if e.g. a citation is for specific subpart of the sentence, then it should be in front of the dot. However in this case I agree.

Comment (RaphaW) [reply paper.tex]: Not sure whether or not max agrees or disagrees with me or you, I would still do it my way

File: paper.tex (hunk 2 above)
Comment (RaphaW) [reply paper.tex]: Oh got it, I thought it was the other way around

File: paper.tex (hunk 7 above)
Comment (EhrenbruderRoschtbraten) [reply paper.tex]: > I disagree. This does depend on what you reference. If the hole statement is based on the cited source it makes sense at the end of the sentence, but if e.g. a citation is for specific subpart of the sentence, then it should be in front of the dot. However in this case I agree.

Yeah, that's true. It was meant like in this case. So @RaphaW in this case it should be after the dot

**Review by 1Maxnet1 — APPROVED** [review]
Looks good, please include @EhrenbruderRoschtbraten 's proposals

**Review by EhrenbruderRoschtbraten — APPROVED** [review]

**Pull request merged**